crates/compiler/parse/src/ast.rs

```diff
@@ -795,7 +795,7 @@ pub enum ValueDef<'a> {
     AnnotatedBody {
         ann_pattern: &'a Loc<Pattern<'a>>,
         ann_type: &'a Loc<TypeAnnotation<'a>>,
-        comment: Option<&'a str>,
+        comment: &'a [CommentOrNewline<'a>],
```

Comment:
I wonder, does it makes sense to name it `spaces` then?

Reply:
Good idea!  Done in 91fdd903e7829ad12e631a1bb454761e23059f7f

Based on the changes in #6809:
https://github.com/roc-lang/roc/blob/e10448e92caf53e136cf1ab8257cfb5bf0e11ec6/crates/compiler/parse/src/expr.rs#L3169-L3171
I decided to name it `spaces_middle`, since that's more specific.

Reply:
Hmm, `spaces` seems off to me, I would expect that to represent an amount of whitespace. How do you feel about `lines_between`?

Reply:
sounds good to me!  done in ddb58a2cd55bf035bf33b03f7a2723b518a36143